models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -296,13 +297,19 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
       close(varfile)
       ncdf4::nc_close(nc)
       
-      #merge nc files
+      #merge nc files of the same year together to enable the assimilation of subannual data
       if(file.exists(file.path(outdir, "previous.nc"))){
         files <- c(file.path(outdir, "previous.nc"), file.path(outdir, "current.nc"))
       }else{
         files <- file.path(outdir, "current.nc")
       }
       mergeNC(files = files, outfile = file.path(outdir, paste(y, "nc", sep = ".")))
+      #The command "cdo" in mergeNC will automatically rename "time_bounds" to "time_bnds". However, "time_bounds" is used 
+      #in read_restart codes later. So we need to read the new NetCDF file and convert the variable name back. 
```

Comment:
This seems like a reasonable fix for this PR. Long term it might be worth being consistent about calling this `time_bnds` throughout, which I think is better aligned with the CF standard: ORNL DAAC [lists](https://daac.ornl.gov/submit/netcdfrequirements/) `time_bnds` as required, and `time_bnds` [appears](http://cfconventions.org/cf-conventions/cf-conventions.html#recording-spacing-original-data)* in the conventions themseves whereas `time_bounds` does not. Can we consider a followup patch to make read_restart use `time_bnds`?

*Though admittedly `time_bnds` only appears in an non-normative example. The CF conventions are messy!

Reply:
Sending this to issue #3303